src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1864,6 +1854,19 @@ private DetectProperties() {
             .setGroups(DetectGroup.BLACKDUCK_SERVER, DetectGroup.BLACKDUCK, DetectGroup.DEFAULT)
             .setDeprecated(USERNAME_PASSWORD_DEPRECATION_MESSAGE, DetectMajorVersion.SEVEN, DetectMajorVersion.EIGHT);
 
+    @Deprecated
+    public static final DetectProperty<NullableStringProperty> DETECT_GRADLE_INSPECTOR_VERSION =
```

Comment:
If the property doesn't do anything, shouldn't it just be removed?

Reply:
Wasn't sure if we do that or deprecate it first then remove.

Reply:
Removing this property since it does not do anything anymore.